homeassistant/components/environment_canada/camera.py

```diff
@@ -1,14 +1,24 @@
 """Support for the Environment Canada radar imagery."""
 from __future__ import annotations
 
+import voluptuous as vol
+
 from homeassistant.components.camera import Camera
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.entity_platform import (
+    AddEntitiesCallback,
+    async_get_current_platform,
+)
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
 from .const import ATTR_OBSERVATION_TIME, DOMAIN
 
+SERVICE_SET_RADAR_TYPE = "set_radar_type"
+SET_RADAR_TYPE_SCHEMA = {
+    vol.Required("radar_type"): vol.In(["Auto", "Rain", "Snow"]),
```

Comment:
Should these be lowercase?
https://github.com/home-assistant/home-assistant.io/pull/22271/files#diff-981e1694d593bd4a0d6a0c72a8970cfc3b9d2cc4691d4ae3b0e9d3539d615481R126

Reply:
Good catch, will change doc.